app/routes/verkiezingen.js

```diff
@@ -0,0 +1,102 @@
+import Route from '@ember/routing/route';
+import { inject as service } from '@ember/service';
+
+export default class VerkiezingenRoute extends Route {
+  @service currentSession;
+  @service store;
+
+  async getInstallatievergaderingenOfBestuurseenheid(bestuurseenheidId) {
+    return await this.store.query('installatievergadering', {
+      'filter[bestuursorgaan-in-tijd][is-tijdsspecialisatie-van][bestuurseenheid][id]':
+        bestuurseenheidId,
+      include: [
+        'status',
+        'bestuursorgaan-in-tijd',
+        'bestuursorgaan-in-tijd.is-tijdsspecialisatie-van',
+      ].join(','),
+    });
+  }
+
+  groupByYearAndStatus(installatievergaderingen) {
+    const yearToStatusIdToIvs = {};
+    const statusIdToStatus = {};
+
+    installatievergaderingen.forEach((vergadering) => {
+      const year = vergadering
+        .get('bestuursorgaanInTijd.bindingStart')
+        .getFullYear();
+      const status = vergadering.get('status');
+      const statusId = status.id;
+      if (!yearToStatusIdToIvs[year]) {
+        yearToStatusIdToIvs[year] = {};
+      }
+      if (!yearToStatusIdToIvs[year][statusId]) {
+        yearToStatusIdToIvs[year][statusId] = [];
+      }
+      yearToStatusIdToIvs[year][statusId].push(vergadering);
+
+      statusIdToStatus[statusId] = status;
+    });
+
+    return { yearToStatusIdToIvs, statusIdToStatus };
+  }
+
+  /*
+  [
```

Comment:
add a comment explaining that this is the format

Reply:
Fixed in next PR